examples/react-web/src/chess/board.js

```diff
@@ -64,6 +67,12 @@ class Board extends React.Component {
 
         {this._getStatus()}
         {disconnected}
+        {this.props.sendChatMessage && this.props.chatMessages && (
+          <Chat
+            onSend={this.props.sendChatMessage}
```

Comment:
Maybe for a future PR: one use case that we might be missing here is the use case that the game itself adds some messages (not by a player)... It would be good if that was available close to the ctx or the G in the game rules, and that if it is a multiplayer game, the server itself could send the message on chat.